bin/_util.js

```diff
@@ -472,5 +506,54 @@ module.exports = {
     get value() {
       return this.parseResolvedValue(this.getResovledValue());
     }
+
+    get valueRef() {
```

Comment:
this getter returns the style-dictionary-compatible string format

https://styledictionary.com/ (see line 158 in Live Demo)